examples/gke/gke.sh

```diff
@@ -204,9 +194,20 @@ kubectl -n cert-manager rollout status --timeout=5m deployment.apps/cert-manager
 echo "Starting the scylla operator..."
 kubectl apply -f ../common/operator.yaml
 
+kubectl wait --for condition=established crd/nodeconfigs.scylla.scylladb.com
 kubectl wait --for condition=established crd/scyllaclusters.scylla.scylladb.com
 wait-for-object-creation scylla-operator deployment.apps/scylla-operator
 kubectl -n scylla-operator rollout status --timeout=5m deployment.apps/scylla-operator
+kubectl -n scylla-operator rollout status --timeout=5m deployment.apps/webhook-server
+
+echo "Configuring nodes..."
+kubectl apply -f nodeconfig-alpha.yaml
+wait-for-object-creation default nodeconfig.scylla.scylladb.com/cluster
+
+# Install local volume provisioner
+echo "Installing local volume provisioner..."
+helm install local-provisioner ../common/provisioner
```

Comment:
let's not mix helm into it

Reply:
might be good to wait for it to get ready

Reply:
> let's not mix helm into it

It's not a change